live-examples/html-examples/inline-text-semantics/css/a.css

```diff
@@ -0,0 +1,4 @@
+a[href^="http"]::before {
+  content: '↑ ';
```

Comment:
I would suggest using the following icon(as SVG) to stay consistent with the rest of MDN:
https://fontawesome.com/icons/external-link-alt?style=solid